jira7/Module.md

```diff
@@ -58,28 +58,31 @@ jira7:JiraConfiguration jiraConfig = {
     }
 };
 
-jira7:Client jiraConnectorEP = new(jiraConfig);
+jira7:Client jiraClient = new(jiraConfig);
 ```
-The `getAllProjectSummaries` function returns the project summary of all the projects.
+
+The `getAllProjectSummaries` remote function returns the project summary of all the projects if successful or an `error` if unsuccessful.
```

Comment:
All these actions seem to be returning `JiraConnectorError` and not `error`, shall we fix all places?
```suggestion
The `getAllProjectSummaries` remote function returns the project summary of all the projects if successful or a `JiraConnectorError` if unsuccessful.
```